src/arviz_plots/plots/khat_plot.py

```diff
@@ -0,0 +1,557 @@
+    x_max = x_flat.max() if x_flat.size else 0.0
+
+    good_k_threshold = hline_values[1] if len(hline_values) > 1 else 0.7
+
+    if backend is None:
+        if plot_collection is None:
+            backend = rcParams["plot.backend"]
+        else:
+            backend = plot_collection.backend
+
+    plot_bknd = import_module(f".backend.{backend}", package="arviz_plots")
+    scalar_ds = xr.Dataset({"pareto_k": xr.DataArray(0)})
+
+    if plot_collection is None:
+        pc_kwargs["figure_kwargs"] = pc_kwargs.get("figure_kwargs", {}).copy()
+        pc_kwargs["figure_kwargs"].setdefault("sharex", True)
+        pc_kwargs["figure_kwargs"].setdefault("sharey", True)
+
+        pc_kwargs["aes"] = pc_kwargs.get("aes", {}).copy()
+
+        if isinstance(color, str) and color in distribution.dims:
+            pc_kwargs["aes"].setdefault("color", [color])
+            color = None
+        elif "model" in distribution.dims:
+            pc_kwargs["aes"].setdefault("color", ["model"])
+
+        pc_kwargs.setdefault("cols", ["__variable__"])
+        pc_kwargs = set_wrap_layout(pc_kwargs, plot_bknd, distribution)
+
+        plot_collection = PlotCollection.wrap(
+            distribution,
+            backend=backend,
+            **pc_kwargs,
+        )
+
+    aes_by_visuals.setdefault("khat", plot_collection.aes_set)
+    aes_by_visuals.setdefault("hlines", [])
+    aes_by_visuals.setdefault("bin_text", [])
+    aes_by_visuals.setdefault("threshold_text", [])
+    aes_by_visuals.setdefault("title", [])
+    aes_by_visuals.setdefault("xlabel", [])
+    aes_by_visuals.setdefault("ylabel", [])
+    aes_by_visuals.setdefault("ticks", [])
+
+    if labeller is None:
+        labeller = BaseLabeller()
+
+    point_rgba = None
+    new_xlim = None
+
+    khat_kwargs = get_visual_kwargs(visuals, "khat")
+    if khat_kwargs is not False:
+        _, khat_aes, khat_ignore = filter_aes(plot_collection, aes_by_visuals, "khat", [])
+
+        if markersize is not None and "size" not in khat_aes:
+            khat_kwargs.setdefault("size", markersize)
+
+        marker_value = khat_kwargs.get("marker")
+
+        if backend == "matplotlib" and "marker" not in khat_aes:
+            marker_value = khat_kwargs.setdefault("marker", "+")
+
+        fillable_marker = marker_value is None or marker_value in _FILLABLE_MARKERS
+        default_color = khat_kwargs.get("color", color)
+
+        if default_color is None and "color" not in khat_aes:
+            default_color = _DEFAULT_POINT_COLOR
+
+        if backend == "matplotlib" and "color" not in khat_aes:
+            base_color = khat_kwargs.pop("color", default_color)
+            point_rgba = alpha_scaled_colors(base_color, khat_data.values, good_k_threshold)
+
+            if fillable_marker:
+                khat_kwargs.setdefault("facecolor", point_rgba)
+                khat_kwargs.setdefault("edgecolor", point_rgba)
+            else:
+                khat_kwargs.setdefault("color", point_rgba)
```

Comment:
all functions that take facecolor and edgecolor also take color as an alias to set both at the same time, so we can skip this check without repercusions